js/zap-common.js

```diff
@@ -61,20 +107,17 @@ function parseYoutubeVideoID(url) {
             var offset = url.indexOf(shortUrlDomain) + shortUrlDomain.length + 1; // Skip over the slash also
             videoInfo.id = url.substring(offset, endPosition);
         }
-        // Assume YouTube video ID string
```

Comment:
No, `parseYoutubeVideoID()` will return a null `videoInfo.id`. I handled this where `parseYoutubeVideoIDI()` is called.